client/app/bundles/shared/components/Footer/Footer.scss

```diff
@@ -0,0 +1,133 @@
+@import '../base';
+
+.footer {
+  width: 100%;
+  height: 450px;
+  background: rgba(0, 0, 0, 0.5);
+  font-family: $font-family;
+
+  @media screen and (max-width: $small) {
+    height: auto;
+  }
+
+  .footer_content {
+    position: relative;
+  }
+
+  .row {
+    display: flex;
+    flex-wrap: wrap;
+    flex-direction: row-reverse;
+    justify-content: space-between;
+    margin-left: 5%;
+    margin-right: 5%;
+
+    @media screen and (max-width: $small) {
+      font-size: 14px;
+      flex-direction: column-reverse;
+      justify-content: space-around;
+      margin-left: 20%;
+    }
+
+    .if_me {
+      text-align: left;
+
+      @media screen and (max-width: $small) {
+        text-align: center;
+      }
+    }
+    
+    .connect {
+      text-align: center;
+    }
+    
+    .resources {
+      text-align: right;
+      margin-right: 30px;
+
+      @media screen and (max-width: $small) {
+        text-align: center;
+        // margin-bottom: 50px;
+      }
+    }
+
+    .table_cell {
+      width: 26%;
+
+      @media screen and (max-width: $small) {
+        width: 75%;
+      }
+
+      
+
+      ul {
```

Comment:
Let's make sure the default left padding isn't there and adjust spacing accordingly! It's especially noticeable when the columns stack on top of one another and the text is slightly off-center.